packages/core/src/builtin-tasks/node.ts

```diff
@@ -31,9 +39,31 @@ subtask('node:start', 'start a ckb node')
       genesisAccountArgs: genesisArgs,
     })
 
-    env.config.network = {
-      url: 'http://localhost:' + port,
+    env.config.network = { url: ckbDockerNetwork.url }
+
+    const builtInDirPath = cachePath('built-in')
+    for (const script of BUILTIN_SCRIPTS) {
+      const filePath = path.join(builtInDirPath, script)
+      if (!fs.existsSync(filePath)) {
+        download(
+          `https://github.com/Daryl-L/ckb-production-scripts/releases/download/scripts/${script}`,
+          builtInDirPath,
+          { filename: script },
+        )
+      }
     }
+    await new Promise((resolve) => setTimeout(resolve, 20000))
```

Comment:
`timersPromises.scheduler.wait(delay[, options])` or `timersPromises.scheduler.yield()` can be used as `sleep`

Ref: https://nodejs.org/api/timers.html#timerspromisesschedulerwaitdelay-options